### PR TITLE
[RFR] Updating provider summary tables fields

### PR DESCRIPTION
**Pull request by tkhamis**
{{ pytest : cfme/tests/containers/test_reload_button_provider.py --use-provider ocp-39-hawk }}
Updating automation after provider summary tables fields were changed .

### CODE REVIEW

**Comment by jawatts** [comment]
:+1: looks good to me.

File: cfme/containers/provider/__init__.py [hunk 1]
```diff
@@ -230,18 +230,18 @@ def num_project(self):
...
         return self._num_db_generic('container_services')
 
     @num_service.variant('ui')
     def num_service_ui(self):
-        if self.appliance.version < "5.7":
-            name = "Services"
-        else:
-            name = "Container Services"
+        name = VersionPick({Version.lowest(): 'Services',
+                            '5.7': 'Container Services'})
```
[the remark this answers is not in the document]
Comment (jawatts) [reply cfme/containers/provider/__init__.py]: I think that was a typo, it should be 5.9

File: cfme/containers/provider/__init__.py [hunk 2]
```diff
@@ -228,18 +228,18 @@ def num_project(self):
     @num_project.variant('ui')
     def num_project_ui(self):
         view = navigate_to(self, "Details")
-        return int(view.entities.summary("Relationships").get_text_of("Projects"))
+        name = VersionPick({Version.lowest(): 'Projects',
+                            '5.9': 'Container Projects'})
```
Comment (quarckster) [inline cfme/containers/provider/__init__.py]: **REQUIRED**
Please just replace`Projects` be `Container Projects`. CFME 5.8 is not supported by the master.

**Comment by jawatts** [comment]
@tkhamis The comment made by @quarckster applies to all of the VersionPick objects, not just the one of line 231

File: cfme/containers/provider/__init__.py [hunk 3]
```diff
@@ -228,18 +228,16 @@ def num_project(self):
...
     def num_service(self):
         return self._num_db_generic('container_services')
 
     @num_service.variant('ui')
     def num_service_ui(self):
-        if self.appliance.version < "5.7":
-            name = "Services"
-        else:
-            name = "Container Services"
+        name = VersionPick({Version.lowest(): 'Services',
```
Comment (mshriver) [inline cfme/containers/provider/__init__.py]: 5.9 is our lowest supported version at the moment, checkout #7600.

You can collapse these to the 5.9  string only, or a version pick for 5.9+

File: cfme/containers/provider/__init__.py [hunk 4]
```diff
@@ -250,7 +248,10 @@ def num_replication_controller(self):
     @num_replication_controller.variant('ui')
     def num_replication_controller_ui(self):
         view = navigate_to(self, "Details")
-        return int(view.entities.summary("Relationships").get_text_of("Replicators"))
+        name = VersionPick({Version.lowest(): 'Replicators',
+                            '5.9': 'Container Replicators'})
+        view = navigate_to(self, "Details")
```
Comment (mshriver) [inline cfme/containers/provider/__init__.py]: this duplicates the navigation on L250

**Pull request merged**